src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -103,6 +103,9 @@ public Startup(IConfiguration configuration, IHostEnvironment hostEnvironment)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+
+            services.AddHealthChecks();
```

Comment:
This is cool!  So do we need to implement something ourselves at /api/health or is there a pre-built one that gives back basic information on this route?  I guess at the very least it's going to error until the API is up and running.

Reply:
Nope, by default, I believe it returns a 200 OK if it's deemed to be in the `healthy state` or a `503 service unavailable` if it's deemed to be in an `unhealthy` state. This is quite a basic implementation but you could take it further by adding more granular health checks for individual services i.e. 

```
using Microsoft.Extensions.Diagnostics.HealthChecks;
using System.Data.SqlClient;
 
namespace HealthCheck.Demo
{
    public static class DbHealthCheckProvider
    {
        public static HealthCheckResult Check(string connectionString)
        {
            try
            {
                using var connection = new SqlConnection(connectionString);
                connection.Open();
                return HealthCheckResult.Healthy();
            }
            catch
            {
                return HealthCheckResult.Unhealthy();
            }
        }
    }
}
```